src/middleware/debug.js

```diff
@@ -10,8 +10,7 @@ export default store => next => action => {
 
   if (isDebug) {
     console.debug('state:', store.getState())
-    self.logSelectors && self.logSelectors()
-    self.logNextReaction && self.logNextReaction()
+    store.doLogEverything && store.doLogEverything()
```

Comment:
I like the idea of the `doLogEverything` but, this switch means that it logs all installed bundles and all action creators after each action, which is just a lot of noise. Especially since those items don't change each time. I think it makes more sense to just do the selectors / next reaction here still.

Reply:
![screen shot 2018-08-03 at 7 40 00 am](https://user-images.githubusercontent.com/67925/43648881-82558892-96f0-11e8-8d88-ff660140b4f6.png)